### ISSUE TITLE
Abominations bug/balance 

### ISSUE THREAD

**Issue by zakhad** [skyarkhangel/Hardcore-SK#1150]
Just a copy&paste from discord as i felt like it warranted being posted as a github issue.

Tribal colony can't reallt deal with early abominations and as such imo I think it's bad gameplay design.

https://www.youtube.com/watch?v=lNGq4JZmVy8&feature=youtu.be As funny as this is, which it is imo, it makes really bad gameplay. I'm only 46 days into my game sitting at about 60-65k total wealth, I have no feasible/legit way of dealing with them unless I kite them to the other side of the map, and even then I'd have to restrict myself to my base, and it's very unlikely that any raid that I encounter will have enough automatics guns capable of dealing with the absurd amount of regeneration, and because they multiply via segmentation by the time I do get a raid with enough to take down maybe 1 abomination there will be like 10-20 on the map.

For the record i have 3 guns, 1 pistol, 2 rifiles that I've scavenged from raids a few bow and arrows but not nearly enough firepower or even the ammunition to deal with them.

I'm not saying it can't be done, I'm saying its not fun, its tedious to an extreme and its bad design.

I'm not sure what i'm asking for really, but could someone come up with a better design or implimentation for them? As random trader npc's falling down and turning into abominations isn't fun. (I would suggest that if its set on fire, all regeneration stops, its how dungeons and dragons deal with trolls for example)

It's not like I know with escape pods I can simply choose not to open them.

Sorry if this came across as whingy, I love this modpack and stuff like this really kills it for me, surely I'm not the only one who thinks like this. Thank you for your time if you've read all of this.

**Comment by Freshpage** [comment]
I agree.  I also don't like the fact that you can just slaughter them in their sleep when you get the message "The faces of the humans contained within the droppods looked terrified beyond belief. They must've experienced something horrifying". This makes them very easy to deal with once you know this.

What I do on Tribal starts is edit C:\Program Files (x86)\Steam\steamapps\common\RimWorld\Mods\ (or wherever your copy is stored) and change <baseBodySize> to 1 and <baseHealthScale> to 3.  

This makes them possible to kill with Tribal weapons, but still a bit of a challenge.

To balance things I have a house rule that I'm not allowed to kill pawns as soon as I get the "The faces of the humans" message, because that is too easy.  I have to let them turn into abominations and then kill them.

**Comment by GhostGum** [comment]
I have had Hardcore SK throw the odd out of my control Certain death scenario at me, of which I am never a fan myself. 
I just recently started a new colony and had an abomination massacre my whole colony, with no viable chance of survival for myself (Mediocre player, but not worth trying to work around the abomination here).
So yeah I loaded and executed the prisoner strait away, and have done so ever since for this playthrough (still early).
